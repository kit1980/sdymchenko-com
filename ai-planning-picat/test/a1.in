10

helloworld
help

helloworld
jello

helloworld
helloeverybody

helloworld
helloworltthere

helloworld
hellogirl

helloworld
helicopter

helloworld
hellboy

helloworld
hellowormhole

helloworld
hallowelt

helloworld
hellothere
