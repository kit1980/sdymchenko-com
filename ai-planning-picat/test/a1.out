<loworld*
*helloworld*
*helloworld*
<<<<<<d*
<<<<world*
*helloworld*
<<<oworld*
<<<<<ld*
*helloworld*
<<<<<world*
